### PR TITLE
Add DOM style attributes in CSP-compliant way

### LINKED ISSUE #1625: BPMN viewer conflicts with strict Content-Security-Policy (CSP) style-src directive
__Describe the Bug__

  

__Steps to Reproduce__

1. Run the BPMN viewer within a web application which has strict `style-src` directives in its Content Security Policy (CSP) header

__Expected Behavior__

Style manipulations should be performed in a way which does not break CSP rules.

__More Information__

See: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src

> the following stylesheets are blocked and won't load [...] Inline style attributes are also blocked [...] As well as styles that are applied in JavaScript by setting the style attribute directly, or by setting `cssText`

The problematic parts of the BPMN code relate to uses of the `domify` helper. When used with inline styles, a strict `style-src` directive will block the styles from taking effect. The styles should be `set directly on the element's style property`, as shown in the above MDN link.

__Environment__

 - Browser: Any of these: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src#browser_compatibility
 - OS: Any
 - Library version: 6.0.0 and up

----

Also reported via [internal conversation](https://camunda.slack.com/archives/CKGH9LR40/p1651476039251279).

### PR DESCRIPTION
**Pull request by WSydnA**
Closes #1625

### CODE REVIEW

File: lib/BaseViewer.js [hunk 1]
```diff
@@ -655,11 +655,9 @@ BaseViewer.prototype._createContainer = function(options) {
 
   var container = domify('<div class="bjs-container"></div>');
 
-  assign(container.style, {
-    width: ensureUnit(options.width),
-    height: ensureUnit(options.height),
-    position: options.position
-  });
+  container.style.width= ensureUnit(options.width);
```
Comment (nikku) [inline lib/BaseViewer.js:658]: Use `applyStyles` here.

Comment (WSydnA) [reply lib/BaseViewer.js]: I've made this change. In doing so I moved the `applyStyles` util into `lib/import/Util.js` to make it more logically accessible to other parts of the library.

**Comment by nikku** [comment]
It was great if we were able to test this in our test suite, but [apparently we cannot](https://github.com/karma-runner/karma/issues/3260).

**Comment by marstamm** [comment]
Hi @WSydnA ,

thank you for your work and bringing it to our attention

We decided to move the `applyStyle` utility to our [`min-dom`](https://github.com/bpmn-io/min-dom) library, so it is reusable through all components. We integrated and merged it in #1645 and I will be closing this PR.

Thanks again for your contribution.

**Pull request closed without merging**